content/manuals/scout/policy/_index.md

```diff
@@ -51,39 +51,47 @@ vulnerabilities. You can use policies to measure and track other aspects of
 supply chain management as well, such as open-source license usage and base
 image up-to-dateness.
 
-## Out-of-the-box policies
+## Policy types
 
-Docker Scout ships the following out-of-the-box policies:
+In Docker Scout, a *policy* is derived from a *policy type*. Policy types are
+templates that define the core parameters of a policy. You can compare policy
+types to classes in object-oriented programming, with each policy acting as an
+instance created from its corresponding policy type.
 
-- [No fixable critical or high vulnerabilities](#no-fixable-critical-or-high-vulnerabilities)
-- [No AGPL v3 licenses](#no-agpl-v3-licenses)
-- [No outdated base images](#no-outdated-base-images)
-- [No high-profile vulnerabilities](#no-high-profile-vulnerabilities)
-- [Supply chain attestations](#supply-chain-attestations)
-- [Default non-root user](#default-non-root-user)
-- [No unapproved base images](#no-unapproved-base-images)
+Docker Scout supports the following policy types:
 
-To give you a head start, Scout enables several policies by default for your
-Scout-enabled repositories. You can customize the default configurations to
-reflect internal requirements and standards. You can also disable a policy
-altogether if it isn't relevant to you. For more information, see [Configure
-policies](./configure.md).
+- [Severity-Based Vulnerability](#severity-based-vulnerability)
```

Comment:
@dvdksn Looks like the policy names are in sentence case on the Scout website, but they are in title case here. Is this intentional?

Reply:
yes, these type names are not published yet

Reply:
Based on [this notion doc](https://www.notion.so/dockerinc/New-policy-names-099709688f16425fb38d0701d8dee28b?pvs=4#e4ea72b842504210a1c53df7344c43ba) some of these do not align. 
For example:
"Severity-Based Vulnerability" -> "Severity-based Vulnerability"
"Up-to-Date Base Images" -> "Up-to-date Base Images"

Haven't checked all.

Reply:
Also can you update anywhere they are shown (titles as well)

Reply:
Oh, just saw [your comment](https://www.notion.so/New-policy-names-099709688f16425fb38d0701d8dee28b?d=10e57a1d4673809a9cfd001c783626d1&pvs=4#e4ea72b842504210a1c53df7344c43ba)